src/main/java/racingcar/RacingService.java

```diff
@@ -0,0 +1,33 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class RacingService {
+    public ArrayList<Car> getCarInfos(Racing racing) {
+        return racing.getCars();
+    }
+
+    public void moveByNumOfTriesAndShowResult(Racing racing) {
+        ResultView resultView = new ResultView();
+        for (int i = 0; i < racing.getNumOfTries(); i++) {
+            move(racing);
+            resultView.showCurrentState(getCarInfos(racing));
+        }
+    }
+
+    public void move(Racing racing) {
+        racing.getCars().stream()
+                .map(car -> {
+                    car.setResultOfRand(getRand(10));
+                    return car;
+                })
+                .filter(car -> car.getResultOfRand() > 4)
```

Comment:
```suggestion
                .filter(car -> car.전진())
```

객체에게 메시지를 던져보면 어떨까요?

Reply:
말씀하신 25 Line에선 rand 결과에 대한 필터링을 한 후, 바로 아래줄(26Line)에서 Car::moveFoward를 사용하고 있습니다
이 구조에 문제가 있나요?